bootstrap/secret/secret.go

```diff
@@ -85,9 +89,11 @@ func NewSecretProvider(
 					secretStoreConfig.RuntimeTokenProvider)
 			}
 
-			secretConfig, err = getSecretConfig(secretStoreConfig, tokenLoader, runtimeTokenLoader, serviceKey, lc)
+			securitySecretTokenDuration := gometrics.NewTimer()
+			secretConfig, err = getSecretConfig(secretStoreConfig, tokenLoader, runtimeTokenLoader, serviceKey, lc, securitySecretTokenDuration)
```

Comment:
```suggestion
			secretConfig, err = getSecretConfig(secretStoreConfig, tokenLoader, runtimeTokenLoader, serviceKey, lc, secureProvider.securityRuntimeSecretTokenDuration)
```

Reply:
Right, I see that now. So the creating of secureProvider.securityRuntimeSecretTokenDuration should be removed from NewSecureProvider, since it will be set here.

Reply:
Leave code as is.